### PR TITLE
Fixed scroll to post from status bar

### PR DESCRIPTION
**Pull request by T-Damer**
- Scroll to button works multiple times now

### CODE REVIEW

File: src/components/ProcessingCard/LinkButton.tsx [hunk 1]
```diff
@@ -25,13 +27,16 @@ export default function ({
...
+      anchor
     >
       <Button
         small
         withArrow
         gradientFont
         type="tertiary"
-        onClick={() => onClick && onClick()}
+        onClick={() => {
+          if (cleanQuery) setLocation(location.split('#')[0])
```
Comment (T-Damer) [inline src/components/ProcessingCard/LinkButton.tsx:38]: 1) Clean the query params
2) Add new query param using `<a>`
3) The hook will catch the change and will scroll to the post

File: src/components/ProcessingCard/LinkButton.tsx [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-  onClick?: () => void
 }
 
-export default function ({
-  url,
-  internal,
-  approved,
-  onClick,
-}: LinkButtonProps) {
+export default function ({ url, internal, approved }: LinkButtonProps) {
```
Comment (T-Damer) [inline src/components/ProcessingCard/LinkButton.tsx:11]: If we'll use this component somewhere else than status, then we can add `onClick`, right now it only used in the top status block

File: src/helpers/flashingPost.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ export default function (element?: HTMLAnchorElement, position?: number) {
 
   const makePostPulse = () => {
     element.classList.toggle('animate-pulse')
-    setTimeout(() => element.classList.toggle('animate-pulse'), 3000)
+    setTimeout(() => element.classList.toggle('animate-pulse'), 4000)
```
Comment (T-Damer) [inline src/helpers/flashingPost.ts:6]: 3000 clips the animation in the middle, 4000 is enough time to play animation two times

File: src/components/ProcessingCard/LinkButton.tsx [hunk 4]
```diff
@@ -1,40 +1,31 @@
...
-      internal={internal}
-      gradientFrom="from-secondary"
-      gradientTo="to-accent"
+    <Button
+      small
+      withArrow
+      gradientFont
+      type="tertiary"
+      onClick={() => {
+        setLocation(url)
```
Comment (T-Damer) [inline src/components/ProcessingCard/LinkButton.tsx:24]: The problem with `Link` wrapping is that location changes after `onClick`, so user need to click twice on the button

**Review by smyzin — APPROVED** [review]
Lgtm

**Pull request merged**